src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -31,7 +33,31 @@ public OverseasEntitySubmissionDtoValidator(EntityNameValidator entityNameValida
     }
 
     public Errors validateFull(OverseasEntitySubmissionDto overseasEntitySubmissionDto, Errors errors, String loggingContext) {
+        String submissionType  = REGISTRATION;
```

Comment:
Couldn't we simplify this a bit? There doesn't seem to be much benefit to the submission type being set. Perhaps just: `if registration
do register validation
else
do update validation`

Reply:
No need for the stings constants and submissionType.

 if (StringUtils.isBlank(overseasEntitySubmissionDto.getEntityNumber())) {
            validateRegistrationDetails(overseasEntitySubmissionDto, errors, loggingContext);
        } else {
            validateUpdateDetails(overseasEntitySubmissionDto, errors, loggingContext);
        }

Reply:
Optional but maybe extract `StringUtils.isBlank(overseasEntitySubmissionDto.getEntityNumber())` into a function called `isRegistration`, so that we know what `StringUtils.isBlank(overseasEntitySubmissionDto.getEntityNumber())` is checking

Reply:
Yea that was why I was explicit initially because I prefer explicit also. isRegistration still sets a boolean and contra previous comments?